db/fake_data.js

```diff
@@ -0,0 +1,51 @@
+const faker = require('faker');
+const db = require('./sequelize');
+
+const listingData = [];
+const availabilityData = [];
+
+function formatDate(date) {
+  const mm = date.getMonth() + 1;
+  const dd = date.getDate();
+
+  return [date.getFullYear(),
+    (mm > 9 ? '' : '0') + mm,
+    (dd > 9 ? '' : '0') + dd,
+  ].join('-');
+}
+
+for (let i = 0; i < 101; i += 1) {
+  const listing = {
+    price: faker.random.number({ min: 30, max: 1000 }),
+    num_reviews: faker.random.number({ min: 0, max: 300 }),
+    avg_rating: faker.random.number({ min: 0, max: 50 }),
```

Comment:
```suggestion
    avg_rating: faker.random.number({ min: 0, max: 5 }),
```
Did you mean 5? I am guessing this is average number of stars from the variable name

Reply:
Yea it does seem confusing, I'll look into it and maybe just change it to a float value.